backend-tests/tests/test_rbac_v2.py

```diff
@@ -1508,3 +1511,601 @@ def test_get_emails_by_group(self, clean_mongo, test_case):
+        release_url = deployments_v2.URL_RELEASE_TAGS.replace("{release_name}", "foo")
+        r = api_client_v2.call("PUT", release_url, test_case["tags"])
+        assert r.status_code == 204
+
+        # get artifacts and releases after tagging
+        api_client.with_auth(test_user.token)
+        api_client_v2.with_auth(test_user.token)
+
+        r = api_client.call("GET", deployments.URL_DEPLOYMENTS_ARTIFACTS)
+        assert r.status_code == 200
+        artifacts = r.json()
+        assert len(artifacts) == test_case["number_of_allwed_releases"]
+        assert artifacts[0]["name"] == "foo"
+
+        r = api_client_v2.call("GET", deployments_v2.URL_RELEASES)
+        assert r.status_code == 200
+        releases = r.json()
+        assert len(releases) == test_case["number_of_allwed_releases"]
+        assert releases[0]["name"] == "foo"
+
+    @pytest.mark.parametrize(
+        "test_case",
+        [
+            {
+                "name": "ok",
+                "use_personal_access_token": False,
+                "user": {
+                    "name": "test1-UUID@example.com",
+                    "pwd": "password",
+                    "roles": ["test"],
+                },
+                "roles": [
+                    {
+                        "name": "test",
+                        "permission_sets_with_scope": [
+                            {
+                                "name": "ReadReleases",
+                                "scope": {"type": "ReleaseTags", "value": ["foo"],},
+                            },
+                            {
+                                "name": "ManageReleases",
+                                "scope": {"type": "ReleaseTags", "value": ["foo"],},
+                            },
+                            {"name": "DeployToDevices",},
+                        ],
+                    },
+                ],
+                "tags": ["foo", "bar"],
+            },
+        ],
+    )
+    def test_manage_releases(self, clean_mongo, test_case):
+        """
+        Tests access to releases with releases scope.
+        """
+        self.logger.info("RUN: %s", test_case["name"])
+
+        uuidv4 = str(uuid.uuid4())
+        tenant, username, password = (
+            "test.mender.io-" + uuidv4,
+            "admin+" + uuidv4 + "@example.com",
+            "secretsecret",
+        )
+        tenant = create_org(tenant, username, password, "enterprise")
+        create_roles(
+            tenant.users[0].token, test_case["roles"], status_code=201,
+        )
+        # create and upload artifacts
+        artifacts = (
+            {"artifact_name": "foo", "device_types": ["arm1"], "size": 256},
+            {"artifact_name": "bar", "device_types": ["arm1"], "size": 256},
+        )
+        api_client = ApiClient(deployments.URL_MGMT)
+        api_client_v2 = ApiClient(deployments_v2.URL_MGMT)
+        api_client.headers = {}  # avoid default Content-Type: application/json
```

Comment:
ok so without that the default will be json Content-type, right?

Reply:
yes; and for uploading artifacts we need different one